collect_app/src/main/java/org/odk/collect/android/widgets/UrlWidget.java

```diff
@@ -105,13 +115,14 @@ protected void onDetachedFromWindow() {
         }
     }
 
-    @Override
-    public void onButtonClick(int buttonId) {
-        if (!isUrlEmpty(stringAnswer)) {
-            customTabHelper.bindCustomTabsService(getContext(), null);
-            customTabHelper.openUri(getContext(), uri);
-        } else {
-            Toast.makeText(getContext(), "No URL set", Toast.LENGTH_SHORT).show();
+    public void onButtonClick() {
+        if (MultiClickGuard.allowClick(QuestionWidget.class.getName())) {
+            if (!isUrlEmpty(stringAnswer)) {
+                customTabHelper.bindCustomTabsService(getContext(), null);
+                customTabHelper.openUri(getContext(), getUri());
+            } else {
+                Toast.makeText(getContext(), "No URL set", Toast.LENGTH_SHORT).show();
```

Comment:
We have `ToastUtils` class which we use for displaying toast so we should use it here as well.